addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -627,6 +627,18 @@ class Configurator extends Component {
                 this.updateBrowserUrl();
             });
         });
+
+        useEffect(() => {
+            const rootEl = document.querySelector(':root');
+            const initalFontSize = rootEl.style.fontSize;
+            rootEl.style.fontSize = '16px';
+            return () => {
+                rootEl.style.fontSize = initalFontSize;
+                if (!rootEl.getAttribute('style')) {
+                    rootEl.removeAttribute('style');
+                }
+            };
+        }, () => []);
```

Comment:
No, instead the CSS has to be adapted I guess. It's also fine that the design changed a little to match a bit more the backend spirit IMO. I think it's not a commit for you to make in the end, a designer should review the configurator design post all refactorings.